src/main.py

```diff
@@ -42,36 +36,41 @@ def render_page():
             p {
                 font-size: 1.5rem;
             }
-            .buttonOn {
+            .button {
                 display: inline-block;
-                background-color: #e7bd3b; 
-                border: none; 
+                background-color: #e7bd3b;
+                border: none;
                 border-radius: 4px;
-                color: white; 
+                color: white;
                 padding: 16px 40px;
-                text-decoration: none; 
-                font-size: 30px; 
-                margin: 2px; 
+                text-decoration: none;
+                font-size: 30px;
+                margin: 2px;
                 cursor: pointer;
             }
-            .buttonOff {
-                background-color: #4286f4;
+
+            .sensor {
+                border: 1px solid black;
             }
+
         </style>
     </head>
     <body>
-        <h1>ESP Web Server</h1> 
-        <p>GPIO state: <strong>""" + gpio_state + """</strong></p>
-        <p>
-            <a href="/?led=on">
-                <button class="buttonOn">ON</button>
-            </a>
-        </p>
+        <h1>ESP Web Server</h1>
+
         <p>
-            <a href="/?led=off">
-                <button class="buttonOn buttonOff">OFF</button>
+            <a href="/?measure=on">
+                <button class="button">MEASURE</button>
```

Comment:
We replace the ON/OFF buttons with a single `MEASURE` button, which will act on all sensors.